waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -436,14 +437,6 @@ when defined(rln) or (not defined(rln) and not defined(rlnzerokit)):
 
     return ok(true)
 
-  proc insertMember*(rlnInstance: RLN[Bn256], idComm: IDCommitment): bool =
```

Comment:
It allows inserting one member at a time, which gives the user the impression that it is supported, where it would inadvertently corrupt their merkle tree

Reply:
What corruption do you mean, can you please elaborate?

Reply:
It gives the impression to the user that inserting one member at a time is allowed, which may lead to them using it to batch updates instead of atomically batching using `insertMembers`. I will add the proc back due to separate functional concerns though, as you've rightly pointed out.

Addressed in 37bd29fb